src/main/plugins/org.dita.pdf2/xsl/fo/topic.xsl

```diff
@@ -1022,8 +1022,8 @@ See the accompanying LICENSE file for applicable license.
                 <xsl:call-template name="get-id"/>
               </xsl:attribute>
             </xsl:if>
-            <xsl:apply-templates select="*[not(contains(@class,' topic/title '))]"/>
-            <xsl:apply-templates select="*[contains(@class,' topic/title ')]"/>
+            <xsl:apply-templates select="*[not(contains(@class,' topic/title ') or contains(@class,' topic/desc '))]"/>
+            <xsl:apply-templates select="*[contains(@class,' topic/title ') or contains(@class,' topic/desc ')]"/>
```

Comment:
the structure of <fig> is title then desc then everything else, so the template needs to deal with both desc and title, not only title separately

ref: https://docs.oasis-open.org/dita/v1.0/langspec/fig.html